src/data/abilities/ab-attrs/add-second-strike-ab-attr.ts

```diff
@@ -44,7 +44,7 @@ export class AddSecondStrikeAbAttr extends PreAttackAbAttr {
         hitCount.value += 1;
       }
 
-      if (multiplier?.value && pokemon.turnData.hitsLeft === 1) {
+      if (multiplier?.value && pokemon.turnData?.hitsLeft === 1) {
```

Comment:
```suggestion
      if (multiplier?.value && pokemon.turnData.hitsLeft === 1) {
```
Looks like there was a merge conflict.